src/System Application/Test/AI Partner/src/AzureOpenAITestPartner.Codeunit.al

```diff
@@ -0,0 +1,161 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Test.AI;
+
+using System.AI;
+using System.Privacy;
+using System.TestLibraries.AI;
+using System.TestLibraries.Environment;
+using System.TestLibraries.Utilities;
+
+codeunit 139025 "Azure OpenAI Test Partner"
+{
+    Subtype = Test;
```

Comment:
There is the HttpClient mock, please block all outbound requests and ensure they are handled.

`TestHttpRequestPolicy = BlockOutboundRequests`

https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-httpclient-mock-outbound-calls

Reply:
There should be no outbound requests that need to actually go to any server.
For AI resources, it is handled by platform and this will not block it afaik.